package.json

```diff
@@ -173,7 +173,7 @@
     "webpack-bundle-analyzer": "^4.9.1"
   },
   "engines": {
-    "node": ">=14.17.0 < 21"
+    "node": ">=14.17.0"
```

Comment:
In prod we use node 18, I left a checkbox to tick when have checked that it works as expected:
- [ ] Checked that deploy works correctly